### DISCUSSION TITLE
continue fetching data from the status of failure and last termination

### DISCUSSION THREAD

**Discussion by hktalent** [projectdiscovery/katana#199] [Q&A]
It is recommended to continue fetching data from the status of failure and last termination

I don't know why the program got stuck. I checked the cpu and memory, and there was no exception

I had to terminate the program manually

But I found that every time, if we start again, the previous data and progress will be a pity

**Comment by ehsandeep** [comment]
@hktalent, please always include examples/steps to reproduce the scenario you are talking about; without examples, it's not possible to start the investigation.

**Comment by hktalent** [comment]
@ehsandeep 

```
katana -d 10000 -jc -kf all -hl -system-chrome  -headless-options '--blink-settings="imagesEnabled=false",--enable-quic="imagesEnabled=false"'  -u "https://waimai.meituan.com" -o meituan.json -j -nc
````
After about 30 minutes, it got stuck

No output, and then I stopped manually

In this case, I hope you can add functions

But when I restart, I continue from the last state
